setup.py

```diff
@@ -64,5 +63,5 @@ def read(*parts):
         "Programming Language :: Python :: 3.9",
         "Topic :: Software Development :: Libraries :: Python Modules",
         "Topic :: System :: Operating System",
-    ],
+    ]
```

Comment:
To keep black happy.

```suggestion
    ],
```